examples/tutorial/gl/cube.py

```diff
@@ -210,8 +209,10 @@ def on_draw(self, event):
                           gl.GL_UNSIGNED_INT, None)
 
     def on_resize(self, event):
-        width, height = event.size
-        gl.glViewport(0, 0, width, height)
+        self._resize(*(event.size + event.physical_size))
+
+    def _resize(self, width, height, physical_width, physical_height):
+        gl.glViewport(0, 0, physical_width, physical_height)
         projection = perspective(35.0, width / float(height), 2.0, 10.0)
```

Comment:
I did not follow the discussions regarding to `on_initialize` and `on_resize` that closely. But should it not be that the backend calls `on_resize` before the first draw call?